.github/workflows/schedule-stale.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  schedule:
+    - cron: "0 16 * * *"
+name: Stale Bot workflow
+jobs:
+  build:
+    name: stale
+    runs-on: ubuntu-latest
+    steps:
+      - name: stale
+        id: stale
+        uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 30
+          days-before-close: 30
+          operations-per-run: 10
```

Comment:
I noticed Gatsby had a much higher operations per run number here, is that something we need to align with?

Reply:
https://github.com/actions/stale/blob/main/action.yml#L39
The default is 30, I set it to 10 so that gives me essentially 10 issues to look at every day for the first 70 or so days (to get through the back log of 700) which I thought was a fairly achievable goal and cuts down on the Email spam
No idea why gatsbys is so high but they also have a custom version of the bot which may have something to do with it

Reply:
Will this look over the same 10 issues every day?

Reply:
I believe how it works is that it will look through all the issues in the order specified until it performs 10 actions or runs out of issues, so it will keep going until it marks 10 issues as stale or closes them